examples/shared-library/pom.xml

```diff
@@ -9,7 +9,15 @@
 	<packaging>jar</packaging>
 	<version>0.0.0-SNAPSHOT</version> <!-- unused -->
 
-	<description>Testing a Jenkinsfile and Helper Script with Spock</description>
+	<description>Testing a Pipeline Shared Library with Spock</description>
```

Comment:
Oops! This name wasn't correct. It's just an example, but let's clean it up as long as we're in here.